lib/stream/feed.rb

```diff
@@ -118,12 +127,30 @@ def following(offset = 0, limit = 25, filter = [])
         "limit" => limit,
         "filter" => filter.join(",")
       }
-      @client.make_request(:get, uri, @signature, params)
+      auth_token = create_jwt_token("follower", "read")
+
+      @client.make_request(:get, uri, auth_token, {}, params)
     end
 
     def unfollow(target_feed_slug, target_user_id)
       uri = "/feed/#{@feed_url}/follows/#{target_feed_slug}:#{target_user_id}/"
-      @client.make_request(:delete, uri, @signature)
+      auth_token = create_jwt_token("follower", "delete")
+
+      @client.make_request(:delete, uri, auth_token, {auth_token: auth_token})
+    end
+
+    private
+
+    def create_jwt_token(resource, action, feed_id=nil, user_id=nil)
```

Comment:
I'm setting the `feed_id` to be an instance variable if the `feed_id` is not provided: https://github.com/GetStream/stream-ruby/pull/50/files#diff-36085f1150dae50e5ff426a2f9b34811R148